components/subject/Summary.tsx

```diff
@@ -8,7 +8,7 @@ import {
   ShieldExclamationIcon,
   UserCircleIcon,
 } from '@heroicons/react/24/solid'
-import { ComponentProps, useState } from 'react'
+import { ComponentProps, useState, type JSX } from 'react';
```

Comment:
The semicolon here is out of the project style.  I think this applies in many place, will just leave this one comment though.

Reply:
oh right, codemod applied these. cleaning up now thanks!